src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -9,6 +9,7 @@
   xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
   xmlns:catalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
   xmlns:glob="clr-namespace:System.Globalization;assembly=mscorlib"
+  AutomationProperties.Name=" "
```

Comment:
Why set this to " "?  Why not "" or something else?

I read through https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.name?view=winrt-26100#windows-ui-xaml-automation-automationproperties-name, and it says:

> For localization reasons, you should avoid hard-coded string values for the Name in XAML.

Reply:
Jaws was reading the full content of this control when entering it. Looking online I found this [Stackoverflow](https://stackoverflow.com/questions/35044345/how-can-i-make-jaws-read-the-contents-of-a-usercontrol-the-same-way-it-does-a-wi) conversation. Setting it to empty was got jaws to stop reading the full content.